ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs

```diff
@@ -236,23 +236,35 @@ public override void OnReactInstanceDispose()
             string responseType,
             CancellationToken token)
         {
+            try
+            {
 #if WINDOWS_UWP
-            var storageFile = await StorageFile.GetFileFromApplicationUriAsync(uri).AsTask().ConfigureAwait(false);
-            var inputStream = await storageFile.OpenReadAsync().AsTask().ConfigureAwait(false);
+                var storageFile = await StorageFile.GetFileFromPathAsync(uri.LocalPath).AsTask().ConfigureAwait(false);
```

Comment:
Why `uri.LocalPath` here and not below?

Reply:
You mean in the non-uwp code? Tbh I don't know from where comes the FileSystem.Current namespace used there.

Reply:
Actually it calls same API internally:
https://github.com/dsplaisted/PCLStorage/blob/367df4961332f3edb5156eb47d64af6b1dcc8de7/src/PCLStorage.WinRT/WinRTFileSystem.cs#L65
I wonder if it would really work with just Uri.ToString

Reply:
This is what's being called in WPF I presume:
https://github.com/dsplaisted/PCLStorage/blob/367df4961332f3edb5156eb47d64af6b1dcc8de7/src/PCLStorage.FileSystem.Desktop/DesktopFileSystem.cs#L61

You can test non-UWP by running Playground.Net46, just edit the index.windows.js file to test the API you want.